components/director/go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/lestrrat-go/jwx v1.2.0
 	github.com/lib/pq v1.10.2
 	github.com/machinebox/graphql v0.2.3-0.20181106130121-3a9253180225
+	github.com/maxbrunsfeld/counterfeiter/v6 v6.4.1
```

Comment:
This was needed because I moved a common package from system-broker. System broker uses counterfeiter for mocks, so I decided to introduce it in director.
We discussed in one of the Retrospectives to start using counterfeiter for new mocks and lazily move older mocks to counterfeiter when needed, because mockery is very fragile.